ChatAPI.Domain/LiveConnections/LiveEvents.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace ChatAPI.Domain.LiveConnections
+{
+	public static class LiveEvents
+	{
+		public static readonly string NewMessage = "new-message";
+		public static readonly string NewFriendRequest = "friend-request";
+	}
```

Comment:
Why "New..."? I think Message and FriendRequest is clear enough. The message and friend request will always be new with no exception.

Reply:
You are the one that said that you want us to be more specific and clear with the names.

Reply:
Ok. I like that,